.changeset/one-bundle.md

```diff
@@ -1,6 +1,6 @@
 ---
-"@bigtest/bundler": patch
-"@bigtest/server": patch
+"@bigtest/bundler": minor
+"@bigtest/server": minor
```

Comment:
Because "@bigtest/server" depends on "@bigtest/bundler" it will bump the patch version and add the changelog without even being part of the changeset.